tobe_blog_web/src/services/UserService.ts

```diff
@@ -20,6 +20,12 @@ export function createUser(data: { firstName: string | undefined; lastName: stri
   });
 }
 
+interface Features {

```

Comment:
Please add a prefix "I" to all interface, suggest to "IUserFeature"

Reply:
I removed the Features, using IUserFeatureDTO which defined in /global/types.ts